### PR TITLE
Update pyregion version to 1.2

### PR DESCRIPTION
**Pull request by bsipocz**

### CODE REVIEW

**Comment by cdeil** [comment]
@bsipocz - There's this error!?
https://travis-ci.org/astropy/conda-channel-astropy/jobs/151746497#L8289

```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/astropy-helpers/r
```

**Comment by mwcraig** [comment]
Weird, usually that means one of the packages depends on astropy-helpers and doesn't bundle and/or doesn't use `--offline` but pyregion is using that.

Traveling until Monday, so won't be much use until then.

File: recipe_templates/pyregion/meta.yaml [hunk 1]
```diff
@@ -31,14 +31,16 @@ source:
 requirements:
   build:
     - python
+    - astropy
     - setuptools
     - pyparsing !=2.0.0
-    - numpy x.x   # pin the numpy version, please...
+    - numpy >1.6
```
Comment (mwcraig) [inline recipe_templates/pyregion/meta.yaml]: I'm confused -- `requirements.yml` says that pyregion has numpy compiled extensions, but if that is the case `numpy x.x` must be listed as dependency. To also include a version restriction do this:

```
- numpy x.x
- numpy >1.6
```

**Comment by bsipocz** [comment]
Thanks @mwcraig. I tried adding all possible dependencies to the recipe, but still getting the same error, so would be great if you could have a look at it.

**Comment by cdeil** [comment]
The builds now passed for 9bdbce6 .

@bsipocz @mwcraig - Merge? Or is some other manual checking needed / desirable before publishing the conda packages?

**Comment by bsipocz** [comment]
@cdeil - the conda package should be available in the next 10-20 minutes.

**Comment by cdeil** [comment]
The OS X build has stalled:
https://travis-ci.org/astropy/conda-channel-astropy/jobs/153360695

Linux and Windows packages are there, but OS X packages for 1.2 are not:
https://anaconda.org/astropy/pyregion/files

@bsipocz - Restart OS X build somehow?

**Comment by bsipocz** [comment]
@cdeil - Build is restarted.

**Comment by mwcraig** [comment]
@bsipocz -- thanks for getting this finished!

**Comment by bsipocz** [comment]
@mwcraig -  I'm sorry that I left so many crap commits in there, but I didn't have my computer with me to do a rebase.

**Comment by mwcraig** [comment]
@bsipocz - no worries, I really don't think anyone will ever read the history (and if it makes you feel better: BruceSherwood/vpython-jupyter#21)

**Pull request merged**